build/Rulesets/Roslyn_BuildRules.ruleset

```diff
@@ -35,16 +36,28 @@
     <Rule Id="CA1720" Action="None" />
     <Rule Id="CA1721" Action="None" />
     <Rule Id="CA1724" Action="None" />
+    <Rule Id="CA1801" Action="None" />
+    <Rule Id="CA1802" Action="None" /> <!-- disabled due to lack of support for string literals -->
+    <Rule Id="CA1804" Action="None" />
+    <Rule Id="CA1806" Action="None" />
+    <Rule Id="CA1812" Action="None" /> <!-- disabled as we create many internal types via reflection-based mechanisms -->
+    <Rule Id="CA1814" Action="None" />
     <Rule Id="CA1815" Action="None" />
+    <Rule Id="CA1821" Action="None" />
```

Comment:
(for example, it flags https://github.com/dotnet/roslyn/blob/dac379741525f359062f8d3133412deedc7e9c1e/src/Interactive/Features/Interactive/Core/InteractiveHost.cs#L227-L230)